lib/broccoli-build-error.js

```diff
@@ -0,0 +1,38 @@
+var path = require('path');
+
+/*
+ * Subclass of `Error`.
+ *
+ * Includes detailed information about the errors from
+ * broccoli plugins, such as error type, code frames,
+ * broccoli node name and package versions.
+ */
+function BroccoliBuildError(originalError, node) {
```

Comment:
the whole repo doesn't use classes and still specifies older versions of `node`; I felt it would make sense to convert everything in one go rather than in one place.